src/ocamlorg_frontend/pages/install.eml

```diff
@@ -137,36 +136,65 @@ Layout.render
                 Run the following in a terminal (either Windows PowerShell or Command Prompt):
               </p>
 
-              <%s! Copy_to_clipboard.small_code_snippet ~id:"install-winget-vs2019buildtools" "winget install Microsoft.VisualStudio.2019.BuildTools --override \"--wait --passive --installPath C:\\VS --addProductLang En-us --add Microsoft.VisualStudio.Workload.VCTools --includeRecommended\"" %>
               <%s! Copy_to_clipboard.small_code_snippet ~id:"install-winget-git" "winget install Git.Git" %>
-              <%s! Copy_to_clipboard.small_code_snippet ~id:"install-winget-dkml" "winget install Diskuv.OCaml" %>
+              <%s! Copy_to_clipboard.small_code_snippet ~id:"install-winget-dkml" "winget install OCaml.opam" %>
 
-              <p>
-                And then in a <strong>NEW</strong> terminal:
-              </p>
+          </li>
 
-              <%s! Copy_to_clipboard.small_code_snippet ~id:"dkml-init-system" "dkml init --system" %>
+          <li>
+            <h3>Initialise opam</h3>
+
+            <p>
+              Now, we are ready to initialise opam, which will create a default
+              <a href="<%s Url.tutorial "opam-switch-introduction" %>">opam switch</a>.
+              An opam switch is an isolated environment for the OCaml compiler
+              and any packages you install.
+            </p>
+
+            <%s! Copy_to_clipboard.small_code_snippet ~id:"init-opam" "opam init" %>
+
+            <p> During <code>opam init</code>, you will be asked about Unix
+            support infrastructure. You can select option 1 to let opam manage
+            its own instance of Cygwin installation. For more details, see  </p>

```

Comment:
Where should we "see"? :)

Reply:
This should ultimately link to 3_01_ocaml_on_windows.md, but that needs updating. I'd change the wording:

```suggestion
            it it for you. For details of more advanced alternatives, see  </p>
```

Reply:
I was going to link the [opam blog post detailing it](https://opam.ocaml.org/blog/opam-2-2-0-windows/). Would that work?

Reply:
Sort of - as it happens that blog post also mentions that the additional configurations (with MSYS2, etc.) will be in a later post!

Reply:
I've linked the opam blog for now. We can update it later, if necessary.